kotlin/src/main/kotlin/io/logto/sdk/core/util/TokenUtils.kt

```diff
@@ -0,0 +1,19 @@
+package io.logto.sdk.core.util
+
+import io.logto.sdk.core.exception.LogtoException
+import io.logto.sdk.core.extension.toIdTokenClaims
+import io.logto.sdk.core.type.IdTokenClaims
+import org.jose4j.base64url.Base64Url
+import org.jose4j.jwt.JwtClaims
+
+object TokenUtils {
+    fun decodeIdToken(token: String): IdTokenClaims {
+        val sections = token.split('.')
+        if (sections.size < 2) {
+            throw LogtoException.DecodingException(LogtoException.Decoding.INVALID_JWT)
+        }
+        val payloadSection = sections[1]
+        val payloadJson = Base64Url.decodeToUtf8String(payloadSection)
```

Comment:
we can use a simple [JWT Consumer](https://bitbucket.org/b_c/jose4j/wiki/JWT%20Examples#markdown-header-producing-and-consuming-a-signed-jwt) to do this instead of manually?

Reply:
We just decode to get the claims and don't need to verify the token.
the JWT Consumer will verify the token, we verify the token once we have gained the token from the backend.

Reply:
![image](https://user-images.githubusercontent.com/14722250/150069876-b25d654a-6807-4862-89a2-2958a822b286.png)
looks like you can consume JWT w/o signature verification. just don't add `.setVerificationKey()` in the chain

Reply:
updated